Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -130,12 +220,50 @@ final class PledgeDescriptionViewController: UIViewController {
       .observeValues { [weak self] in
         self?.presentHelpWebViewController(with: .trust)
       }
+
+    self.viewModel.outputs.configureRewardCardViewWithData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        guard let self = self else { return }
+        self.rewardCardContainerView.configure(with: data)
+      }
+
+    self.viewModel.outputs.popViewController
+      .observeForUI()
+      .observeValues { [weak self] in
+        guard let self = self else { return }
+        self.navigationController?.popViewController(animated: true)
+      }
   }
 
   // MARK: - Configuration
 
-  internal func configureWith(value: Reward) {
-    self.viewModel.inputs.configureWith(reward: value)
+  internal func configureWith(value: (project: Project, reward: Reward)) {
+    self.viewModel.inputs.configureWith(data: value)
+  }
+
+  // MARK: - Depress Transform
+
+  @objc func depress(_ gestureRecognizer: UILongPressGestureRecognizer) {
+    let animator = UIViewPropertyAnimator(
+      duration: CheckoutConstants.RewardCard.Transition.DepressAnimation.duration,
+      curve: .linear
+    ) {
+      let transform: CGAffineTransform
+      switch gestureRecognizer.state {
+      case .changed:
+        return
+      case .began:
+        let scale = CheckoutConstants.RewardCard.Transition.DepressAnimation.scaleFactor
+        transform = CGAffineTransform(scaleX: scale, y: scale)
+      default:
+        transform = .identity
+      }
+
+      self.rewardCardContainerShadowView.transform = transform
```

Comment:
Can we use `[weak self]` here?

Reply:
Hmm, don't think we need to - there shouldn't be a retain cycle because the animator is not being retained by the VC